Integrations/integration-Okta.yml

```diff
@@ -0,0 +1,509 @@
+description: Okta integration
+detaileddescription: |-
+  To get an API token go to the following URL:
+  http://developer.okta.com/docs/api/getting_started/getting_a_token.html
+configuration:
+- display: okta URL https://<domain.okta.com>
+  name: url
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: API Token
+  name: apitoken
+  defaultvalue: ""
+  type: 4
+  required: true
+script:
+  script: |-
+    var profileArgs = [
+        'firstName',
+        'lastName',
+        'email',
+        'login',
+        'secondEmail',
+        'middleName',
+        'honorificPrefix',
+        'honorificSuffix',
+        'title',
+        'displayName',
+        'nickName',
+        'profileUrl',
+        'primaryPhone',
+        'mobilePhone',
+        'streetAddress',
+        'city',
+        'state',
+        'zipCode',
+        'countryCode',
+        'postalAddress',
+        'preferredLanguage    ',
+        'locale',
+        'timezone    ',
+        'userType',
+        'employeeNumber',
+        'costCenter',
+        'organization',
+        'division',
+        'department',
+        'managerId',
+        'manager'
+    ];
+
+    var fixUrl = function(base) {
+        res = base;
+        if (base && base[base.length - 1] != '/') {
+            res = res + '/';
+        }
+        return res;
+    };
+
+    var parseResponse = function(resp) {
+        if (resp.StatusCode === 200) {
```

Comment:
Maybe should also be in top level and get the parsing func (JSON.parse in this case)